test/unit/notify/send-email.test.js

```diff
@@ -10,24 +10,26 @@ let sendEmail
 let emailAddress
 let defaultEmailAddress
 let defaultReference
-
 describe('send email', () => {
   beforeEach(() => {
     jest.clearAllMocks()
     jest.resetModules()
-
+    
     jest.mock('../../../ffc-pay-alerts/notify/validate-email')
     validateEmail = require('../../../ffc-pay-alerts/notify/validate-email')
 
     notifyClient = require('notifications-node-client').NotifyClient
     jest.mock('notifications-node-client')
 
     sendEmail = require('../../../ffc-pay-alerts/notify/send-email')
-
+    sendEmails = require('../../../ffc-pay-alerts/notify/send-emails')
```

Comment:
Not needed. This is a unit test file for `sendEmail` only its dependencies should be imported